src/index.test.ts

```diff
@@ -1715,3 +1720,438 @@ describe('READY event emission', () => {
         expect(client.emit).toHaveBeenCalledWith(EVENTS.READY);
     });
 });
+
+describe('Experimental options togglesStorageTTL disabled', () => {
+    const lastUpdateStorageKey = 'repoLastUpdateTimestamp';
+
+    describe('Handling last update flag storage', () => {
+        let storageProvider: IStorageProvider;
+        let saveSpy: jest.SpyInstance;
+
+        class Store implements IStorageProvider {
+            public async save() {
+                return Promise.resolve();
+            }
+
+            public async get() {
+                return Promise.resolve([]);
+            }
+        }
+
+        beforeEach(() => {
+            storageProvider = new Store();
+            saveSpy = jest.spyOn(storageProvider, 'save');
+            jest.clearAllMocks();
+        });
+
+        test('Should not store last update flag when fetch is successful', async () => {
+            fetchMock.mockResponseOnce(JSON.stringify(data));
+
+            const config: IConfig = {
+                url: 'http://localhost/test',
+                clientKey: '12',
+                appName: 'web',
+                storageProvider,
+                experimental: {
+                },
+            };
+
+            const client = new UnleashClient(config);
+            await client.start();
+            expect(saveSpy).not.toHaveBeenCalledWith(
```

Comment:
On devrait tester plus "large"
```suggestion
            expect(saveSpy).not.toHaveBeenCalledWith(lastUpdateStorageKey, expect.anything())
```

Reply:
ha oui ok juste parce que c'est un not .. effectivement c'est l'appel a la clé qui nous interesse :)